### ISSUE TITLE
Example tests don't run

### ISSUE THREAD

**Issue by DenisKusakin** [jdi-templates/jdi-light-junit-template#7]
The following exception occurred when trying to run example tests

```
java.lang.RuntimeException: Can't get WebDriver: chrome

	at com.epam.jdi.light.common.Exceptions.exception(Exceptions.java:31)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriver(WebDriverFactory.java:210)
	at org.mytests.tests.example.extensions.JDISetupExtension.beforeAll(JDISetupExtension.java:30)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$7(ClassBasedTestDescriptor.java:359)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:359)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:189)
	at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:78)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:132)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:125)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:135)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:123)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:122)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:80)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:229)
	at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:197)
	at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:211)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:191)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:221)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.RuntimeException: java.lang.RuntimeException: Failed to Run driver 'chrome' from driverPath = null
	at com.epam.jdi.tools.func.JFunc1.execute(JFunc1.java:21)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriver(WebDriverFactory.java:208)
	... 36 more
Caused by: java.lang.RuntimeException: Failed to Run driver 'chrome' from driverPath = null
	at com.epam.jdi.light.common.Exceptions.exception(Exceptions.java:31)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriverException(WebDriverFactory.java:135)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriverByName(WebDriverFactory.java:115)
	at com.epam.jdi.tools.func.JFunc1.execute(JFunc1.java:19)
	... 37 more
Caused by: java.lang.RuntimeException: Can't run and register new driver 'chrome'
	at com.epam.jdi.light.common.Exceptions.exception(Exceptions.java:31)
	at com.epam.jdi.light.driver.WebDriverFactory.registerNewDriver(WebDriverFactory.java:82)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriverFromName(WebDriverFactory.java:146)
	at com.epam.jdi.light.driver.WebDriverFactory.getDriverByName(WebDriverFactory.java:113)
	... 38 more
Caused by: java.lang.RuntimeException: java.lang.NullPointerException
	at com.epam.jdi.tools.func.JFunc1.execute(JFunc1.java:21)
	at com.epam.jdi.light.driver.WebDriverFactory.registerNewDriver(WebDriverFactory.java:78)
	... 40 more
Caused by: java.lang.NullPointerException
	at com.epam.jdi.light.driver.get.DriverData.driverSettings(DriverData.java:95)
	at com.epam.jdi.tools.func.JFunc1.execute(JFunc1.java:19)
	... 41 more
```

As far as I can see, this could be fixed by changing call order a bit in **JDISetupExtension**
**Current**: 
```
WebDriverFactory.getDriver("chrome");
initSite(SiteJdi.class);
```
**The order that worked for me**:
```
initSite(SiteJdi.class);
WebDriverFactory.getDriver("chrome");
```

**Comment by pnatashap** [comment]
initSite should be the first, you a correct.
Also it is better to use WebPage.openSite instead of this two lines

**Comment by pnatashap** [comment]
Issue was fixed in JDI version 1.4.6
